packages/client/src/components/Cards/Card.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Shouldn't the path be `/components/cards/`?

Reply:
Cards is plural this is a single Card, we might have something else in the folder, I didn't want to use index to re export everything, I didn't bother

Reply:
Are we talking about the same thing? I am talking about changing the `Cards` directory to `cards`.

Reply:
oh I see, mmm usually components name are uppercase not sure and plus it will messup with git on my mac if we do this change ;-p let's keep it as it is for now
![image](https://user-images.githubusercontent.com/2563196/101338734-48e18380-3875-11eb-9d0c-332522c27834.png)